content/blog/open-source-monitoring-at-the-hpe-customer-technology-center-böblingen.md

```diff
@@ -0,0 +1,377 @@
+
+First, the exporter is reading the input data, i.e. the ILO ip addresses and the ILO username and password. Next the exporter is retrieving the correct URLs for each monitored server and store it in a Python directory.  Afterwards, the script starts the http server and the corresponding Prometheus counters and gauges. Having completed these preparation steps, the script enters an endless loop that first captures the start time of the current iteration, second retrieves the data from the monitored server, third displays the captured data and waits until the current monitoring intervall is completed. Additionally, I have added that at least once per day the input data and monitor URLs are refreshed. This allowed me to change the list of monitored systems without starting and stopping the script. I just needed to edit the input data file to adjust the list of monitored systems.
+
+## Main routine
+
+As a result, the Python main routine of the ILO Prometheus exporter is as simple as (I removed some of the log commands to improve the readability):
```

Comment:
I would add a few words after "as". For example, "As a result, the Python main routine of the ILO Prometheus exporter is as simple as shown below (I removed some of the log commands to improve the readability):"